### DISCUSSION TITLE
Control of the send button according to the form data

### DISCUSSION THREAD

**Discussion by sbc640964** [filamentphp/filament#7983] [Help]
I want to control the submit button according to the form data but I notice that $get will not work in this case.

```php
Action::make('create_new')
  ->steps(...)
  ->action(...)
  ->modalSubmitAction(function(Get $get){
    $get('same-field'); // error
  });
```

the error:
```
Typed property Filament\Forms\Components\Component::$container must not be accessed before initialization
```

**Comment by danharrin** [comment]
What are you trying to achieve

**Reply by sbc640964** [reply]
Make the button disabled according to certain parameters in the form
I use the MountableAction and no StaticAction

**Reply by danharrin** [reply]
Isnt that the job for validation?

**Reply by sbc640964** [reply]
I need to verify against credit clearing in the form, only after I have the returned token I want to allow the form to be sent,

And since it depends on the selection field of the clearance, I would like to display a dynamic button depending on the clearance type

For one of the cashiers I need a non-static button and for the other I can also use a static one but it must be disabled until verification with the credit company.
